js-functions/wk1 submission.js

```diff
@@ -0,0 +1,40 @@
+/*
+Example x = 'abcde';
+1. Write a JavaScript function that reverse a string.
+Expected Output : 'edcba'
+Click me to see the solution
+
+2. write a function that returns a grade based on the followin conditions
+
+
+If above 90, then A .
+else If above 80, then B .
+else If above 70, then C .
+else If above 60, then D .
+else F .
+
+*/
+
+function reverse(str){
+  var newString = ''
```

Comment:
use `let`. 

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var